app/src/main/res/values/strings.xml

```diff
@@ -87,26 +87,26 @@
     <string name="cancel">Cancel</string>
     <string name="ok">OK</string>
     <string name="connect_wifi">Connect</string>
-    <string name="connection_error_simple">Can\'t connect to network.</string>
-    <string name="connection_internal_error">Can\'t connect: Internal Error!</string>
-    <string name="connection_app_disallowed">This app is not allowed to add Wi-Fi networks. Enable permission in Settings.</string>
-    <string name="connection_duplicate">Can\'t connect: Duplicate network found.</string>
-    <string name="connection_app_exceed_limit">App exceeded maximum Wi-Fi suggestion limit. Re-install app to reset.</string>
-    <string name="connection_ntwk_invalid">Can\'t connect: Network invalid.</string>
-    <string name="connection_ntwk_disallowed">Can\'t connect: Network disallowed.</string>
-    <string name="ssid">Wi-Fi SSID</string>
+    <string name="connection_error_simple">Can\'t connect to the network.</string>
+    <string name="connection_internal_error">Can\'t connect due to an internal error!</string>
+    <string name="connection_app_disallowed">Grant this app permission to add Wi-Fi networks in the system settings.</string>
+    <string name="connection_duplicate">Can\'t connect because a duplicate network was found.</string>
+    <string name="connection_app_exceed_limit">The app exceeded the maximum Wi-Fi suggestion limit. Re-install it to reset the limit.</string>
+    <string name="connection_ntwk_invalid">Can\'t connect to an invalid network.</string>
+    <string name="connection_ntwk_disallowed">Can\'t connect since this is not an allowed network.</string>
+    <string name="ssid">Wi-Fi network name</string>
     <string name="password">Wi-Fi Password</string>
-    <string name="copy_ssid">Copy SSID</string>
+    <string name="copy_ssid">Copy Wi-Fi network name</string>
     <string name="copy_password">Copy password</string>
     <string name="wifi">Wi-Fi</string>
     <string name="encryption_type">Encryption type</string>
     <string name="open_network">Open</string>
     <string name="wifi_password_hint">Password (leave blank if open)</string>
-    <string name="wifi_ssid_hint">Name/SSID</string>
+    <string name="wifi_ssid_hint">Network name/SSID</string>
     <string name="wifi_enctype_hint">Encryption type:</string>
     <string name="wep">WEP</string>
     <string name="wpa">WPA</string>
-    <string name="open_source_licenses">Open-source licenses</string>
+    <string name="open_source_licenses">Libre licenses</string>
```

Comment:
I think Open-source licenses should be ok. Samsung and Google(in YouTube) also use "Open source licenses", and quite a few other apps also use similar wording(Zoom uses "Open source software" and Firefox uses "Licensing information"). Although "Libre licenses" is not wrong, it would require changing the other translations as well. Thanks for understanding.

View the screenshots:
[YouTube](https://user-images.githubusercontent.com/58442255/162123582-0c0a38ef-19d6-4019-90c3-c5b226a9255f.png)
[Samsung Gallery](https://user-images.githubusercontent.com/58442255/162123722-cb554782-f6c4-42b7-9307-07020716faf8.png)
Thanks